inc/templates/edit-form.tpl.php

```diff
@@ -52,7 +52,9 @@
 <script type="text/html" id="tmpl-shortcode-ui-field-textarea">
 	<div class="field-block">
 		<label for="{{ data.attr }}">{{ data.label }}</label>
-		<textarea name="{{ data.attr }}" id="{{ data.attr }}" placeholder="{{ data.placeholder }}">{{ data.value }}</textarea>
+		<textarea name="<#if ( data.attr ) { #>{{ data.attr }}<#}else{#>inner_content<#}#>" 
```

Comment:
the whole purpose of adding `inner_content` is to not use it as attribute. That was the reason i added a new template for `inner_content`. Should I revert back to adding a new template ?